faqs/_posts/2016-03-30-what-are-xls-reports.md

```diff
@@ -0,0 +1,140 @@
+
+In our case, the filter function is introduced on the [Mali Airport XLS Template](https://docs.google.com/a/ona.io/spreadsheets/d/16Lt-FgO-M0NwlNysQlVN_kw0y7__5242BocIL5ZBNsM/edit?usp=sharing) only to display airports which are 1000 ft above ground level.
+
+<br>
+![](/content/screenshots/faq/xls-reports-3.png)
+
+`<jx:forEach items="${data}"var=“airport”select=“${airport.deviation_ft>1000}”>`
+
+The line of code above invokes that from the data named ‘d’, you only retrieve elevation feet that is above 1000.
+
+`${airport.iso_region}|${airport.municipality}|${airport.ident}|${airport.name}|${airport.type}|${airport.deviation_ft}`
+
+The line of code above invokes that you retrieve information about region, municipality, code, name, type and elevation feet. It is important to note that all the information retrieved will be subjected to our filter.
+
+Below is an example of the report that is generated from the above template:
+
+<br>
+![](/content/screenshots/faq/xls-reports-4.png)
+
+>
+**Note:** <br/> All the elevation feet displayed in the above example are over 1000 feet.
+
+### How do I do summation on my report?
+
+You can sum up values in a report by introducing a summation function on an XLS Template.
+<br>
+![](/content/screenshots/faq/xls-reports-5.png)
+<br>
+The Excel formula introduced above will sum up the elevation feet values that will be generated in the report.
+<br><
+ ![](/content/screenshots/faq/xls-reports-6.png)
+
+### How can I generate charts or graphs in an XLS Report?
+
+You can generate graphical representations on an XLS Report provided you introduce an Excel function into the XLS Template that will capture what you intend to graphically display.
+
+You can display a pie chart to show the distribution between the two types of airports in Mali (small airport and medium airport)
+
+1. Create a new worksheet on your template and call it calculations. This will hold the formula for getting the distribution between the two airports.
+<br><br>
+![](/content/screenshots/faq/xls-reports-7.png)
+2. Insert a new worksheet and call it dashboard. It will hold our blank chart.
+<br><br>
+![](/content/screenshots/faq/xls-reports-8.png)
+<br><br>
+3. You'll generate the output below. On the calculation sheet you'll notice that when you click on a result (e.g 26) the formula being displayed is similar to the one we introduced in our XLS Template.
+<br><br>
+![](/content/screenshots/faq/xls-reports-9.png)
+<br><br>
+4. When you switch to the dashboard sheet of the report you will view the following output.
+<br><br>
+![](/content/screenshots/faq/xls-reports-10.png)
+
+### Where do I upload my XLS Report Template?
+
+Once your XLS Report Template is authored, follow the steps below to upload your template to your form on Ona:
```

Comment:
You can remove "on Ona" from the ends of all of these sentences, since it's implicit.